Description: Debianize the package
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2016-09-14

--- python-xstatic-objectpath-1.2.1.0.orig/xstatic/pkg/objectpath/__init__.py
+++ python-xstatic-objectpath-1.2.1.0/xstatic/pkg/objectpath/__init__.py
@@ -34,9 +34,9 @@ HOMEPAGE = 'https://github.com/mike-marc
 LICENSE = 'MIT'
 
 from os.path import join, dirname
-BASE_DIR = join(dirname(__file__), 'data')
+#BASE_DIR = join(dirname(__file__), 'data')
 # linux package maintainers just can point to their file locations like this:
-#BASE_DIR = '/usr/share/javascript/angular-schema-form'
+BASE_DIR = '/usr/share/javascript/objectpath'
 
 LOCATIONS = {
     # CDN locations (if no public CDN exists, use an empty dict)
